services/external-devices/src/main/java/org/opfab/externaldevices/services/DevicesService.java

```diff
@@ -93,14 +90,21 @@ public void sendSignalForUser(String opFabSignalKey, String userLogin) throws Ex
 
     @Scheduled(cron = "${operatorfabric.externaldevices.watchdog.cron:*/5 * * * * *}", zone = "UTC")
     public void sendWatchdog() {
-        if(externalDevicesWatchdogProperties.getEnabled()) {
+        if (externalDevicesWatchdogProperties.getEnabled()) {
+
+            Map<String, Boolean> hostAndPortMap = new HashMap<>();
```

Comment:
use HashSet which is more adapted in this case (no need to store a Boolean)

Reply:
Done.